.github/workflows/_test_container.yml

```
@@ -0,0 +1,124 @@
+          mode: ${{ inputs.build_mode }}
+          test_tags: ${{ inputs.test_tags }}
+          test_coverage: ${{ inputs.test_coverage }}
+          binary_output_path: ${{ inputs.test_working_dir }}/${{ inputs.binary_output_path }}
+
+      - name: Run tests
+        shell: bash
+        working-directory: ${{ inputs.test_working_dir }}
+        run: |
+          # Verify binary works before running tests
+          ${{ inputs.binary_output_path }} version
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)